src/test/java/com/rultor/agents/github/qtn/QnReleaseTest.java

```diff
@@ -77,4 +79,48 @@ public void buildsRequest() throws Exception {
         );
     }
 
+    /**
+     * QnRelease can deny release when tag is outdated.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void denyOutdatedTag() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        repo.releases().create("1.7");
+        issue.comments().post("release `1.6`");
+        MatcherAssert.assertThat(
+            new QnRelease().understand(
+                new Comment.Smart(issue.comments().get(1)), new URI("#")
+            ),
+            Matchers.is(Req.EMPTY)
+        );
+        MatcherAssert.assertThat(
+            new Comment.Smart(issue.comments().get(2)).body(),
+            Matchers.containsString("There is already a release `1.7`")
+        );
+        issue.comments().post("release");
+    }
+
+    /**
+     * QnRelease can deny release when tag name is not given.
```

Comment:
@original-brownbear I see your point, but on the other hand we have this principle... http://www.yegor256.com/2015/06/22/valid-reasons-to-reject-bug-fix.html#it-solves-more-than-one-issue